global.yaml

```diff
@@ -464,6 +464,15 @@ global:
   # will ignore the helm test yaml files when generating the template
   enableHelmTest: false
 
+  # Configures DNS certificates provisioned through Chiron linked into Pilot.
+  # Te values in this file are all hard-coded; please ensure the namespaces
+  # in dnsNames are consistent with those of your services.
+  certificates:
+    - secretName: dns.istio-galley-service-account
+      dnsNames: [istio-galley.istio-system.svc, istio-galley.istio-system]
```

Comment:
Any way to parameterize these DNS names with the `istioNamespace` setting?

Reply:
I have the same question; all the values in this file are hard-coded.
Add @costinm and @howardjohn regarding to this question.

Reply:
Not that I know of unfortunately